_includes/socials.html

```diff
@@ -0,0 +1,19 @@
+<div class="social-badges">
+  <div class="social-badge"> <!-- GitHub -->
+    <!-- GitHub -->
+    <a class="github-button" href="https://github.com/apache/arrow" data-size="large" data-show-count="true" aria-label="Star apache/arrow on GitHub">Star</a>
+  </div>
+  <style type="text/css">
+    .custom-social-badge{display:inline;padding:4px}.custom-social-badge a{text-decoration:none;outline:0}.custom-social-badge .widget{display:inline-block;overflow:hidden;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif;font-size:0;line-height:0;white-space:nowrap}.custom-social-badge .btn:not(:last-child){border-radius:.25em 0 0 .25em}.custom-social-badge .widget-lg .btn{height:28px;padding:5px 10px;font-size:12px;line-height:16px}.custom-social-badge .btn{border-radius:.25em;color:#25292e;background-color:#ebf0f4;border-color:#d1d9e0;background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3clinearGradient id='o' x2='0' y2='1'%3e%3cstop stop-color='%23f6f8fa'/%3e%3cstop offset='90%25' stop-color='%23ebf0f4'/%3e%3c/linearGradient%3e%3crect width='100%25' height='100%25' fill='url(%23o)'/%3e%3c/svg%3e");background-image:-moz-linear-gradient(top,#f6f8fa,#ebf0f4 90%);background-image:linear-gradient(180deg,#f6f8fa,#ebf0f4 90%);filter:progid:DXImageTransform.Microsoft.Gradient(startColorstr='#FFF6F8FA',endColorstr='#FFEAEFF3');filter:none;position:relative;display:inline-block;display:inline-flex;height:14px;padding:2px 5px;font-size:11px;font-weight:600;line-height:14px;vertical-align:bottom;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-repeat:repeat-x;background-position:-1px -1px;background-size:110% 110%;border:1px solid}.custom-social-badge .btn:hover,.custom-social-badge .btn:focus{background-color:#e5eaee;background-position:0 -.5em;border-color:#d1d9e0;background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg'%3e%3clinearGradient id='o' x2='0' y2='1'%3e%3cstop stop-color='%23eff2f5'/%3e%3cstop offset='90%25' stop-color='%23e5eaee'/%3e%3c/linearGradient%3e%3crect width='100%25' height='100%25' fill='url(%23o)'/%3e%3c/svg%3e");background-image:-moz-linear-gradient(top,#eff2f5,#e5eaee 90%);background-image:linear-gradient(180deg,#eff2f5,#e5eaee 90%);filter:progid:DXImageTransform.Microsoft.Gradient(startColorstr='#FFEFF2F5',endColorstr='#FFE4E9ED')}.custom-social-badge .btn:hover,.custom-social-badge .btn:focus{filter:none}.custom-social-badge .btn:active{background-color:#e6eaef;border-color:#d1d9e0;background-image:none;filter:none}
```

Comment:
...yikes.

Since this was presumably taken from the Github button, do we know what the license of the Github button's source is?

Reply:
It's from https://github.com/buttons/github-buttons. It's BSD 2-Clause. This is a very small piece with changes made to it so we are likely in fair-use territory but I went ahead and added the required notices.

Reply:
P.S. I opened an issue in that repo to see if the developer would be willing to add support for user-defined SVG icons in the buttons, which would allow this all to be implemented much more cleanly. But the developer kindly explained that it was not feasible and closed the issue.

Reply:
Another route to doing this more cleanly is to convince the folks at GitHub to add social icons to their [Octicons](https://primer.style/foundations/icons) icon set, which the github-buttons project uses. I opened an issue for that but it will likely be a long wait if it happens at all, and the github-buttons developer would then have to pull the new social icons into github-buttons. I'll keep pushing on that and if it happens I will return here and reimplement this fix more cleanly.

Reply:
I wonder if there's an alternative social buttons set we can use.

Reply:
I looked around a bit and couldn't find one that would display the GitHub star count. If anyone else is reading this and knows a good, clean, reliable solution for this, please share.